community/javascript/remove-unused-vars/transform.ts

```diff
@@ -0,0 +1,13 @@
+import { FileInfo } from 'jscodeshift';
+// @ts-expect-error
+import putout from 'putout';
+// @ts-expect-error
+import removeUnusedVars from '@putout/plugin-remove-unused-variables';
+
+export default function transformer(file: FileInfo) {
+  const output = putout(file.source, {
+    plugins: [removeUnusedVars],
+  });
+
+  return output.code;
```

Comment:
@coderaiser You'll probably be able to spot the issue instantly, but I keep running into this:

<img width="754" alt="image" src="https://user-images.githubusercontent.com/3030010/237017631-4b7ff14d-e134-4cb1-bec6-0ca141a40ca5.png">